packages/webchat/test/tsconfig.json

```diff
@@ -0,0 +1,18 @@
+{
+  "extends": "../../../tsconfig.packages.json",
+  "compilerOptions": {
+    "noEmit": true,
```

Comment:
Is noEmit needed? We don't have it in the other test tsconfig

Reply:
Not sure if it is needed, but, again this is a recommended step by the Cypress documentation